website/snippets/_relative-path.md

```diff
@@ -0,0 +1 @@
+<span>Paths specified in <code>{props.path}</code> must be relative to the location of your `dbt_project.yml` file. Avoid using absolute paths like <code>{props.absolute}</code>, as they may lead to unexpected behavior.</span>
```

Comment:
If we're telling them it "must" be something, can we be specific about what will break?